packages/compass-shell/src/components/shell-header/shell-header.jsx

```diff
@@ -130,7 +136,9 @@ export class ShellHeader extends Component {
             &gt;_MONGOSH
             {!isExpanded && isOperationInProgress && (
               <span className={operationInProgressStyles}>
-                <ShellLoader size="12px" />
+                <div className={inProgressSpinLoaderStyles}>
+                  <SpinLoader size="12px" />
+                </div>
```

Comment:
Yeah, good point. I did base this off [what we do in browser-repl](https://github.com/mongodb-js/mongosh/blob/9571485045118908f3a55b804f978dea7674797b/packages/browser-repl/src/components/shell-loader.tsx#L5-L7) itself, but this seems worth fixing; I’ll do this in a separate PR though to avoid dragging this one here on much longer.